node/tests/integration/main.rs

```diff
@@ -571,6 +538,30 @@ async fn assert_slot_status_call(
     );
 }
 
+async fn assert_slot_status_fold(
+    client: &AzeClient,
+    account_id: AccountId,
+    slot_data: GameStorageSlotData
+) {
+    let (account, _) = client.get_account(account_id).unwrap();
+    let game_account_storage = account.storage();
+
+    let fold_index = slot_data.current_turn_index() + 9;
+
+    // check is_fold
+    assert_eq!(
+        game_account_storage.get_item(fold_index),
+        RpoDigest::new([Felt::from(1 as u8), Felt::ZERO, Felt::ZERO, Felt::ZERO])
+    );
+
+    let next_turn_index = slot_data.current_turn_index() + 13;
+    // check next turn index
+    assert_eq!(
+        game_account_storage.get_item(CURRENT_TURN_INDEX_SLOT),
+        RpoDigest::new([Felt::from(next_turn_index), Felt::ZERO, Felt::ZERO, Felt::ZERO])
+    );
+}
+
 async fn fund_account(client: &mut AzeClient, account_id: AccountId, faucet_account_id: AccountId) {

```

Comment:
why are we minting two notes here to same account id ?

Reply:
This func is also used by test_cards_dist, where two notes need to be minted as there are two players to whom cards are being distributed.